internal/cfgplugins/policyforwarding.go

```diff
@@ -370,6 +474,78 @@ func PolicyForwardingConfig(t *testing.T, dut *ondatra.DUTDevice, traffictype st
 	}
 }
 
+// PolicyForwardingConfigScale configures policy forwarding using multiple traffic-policies.
+// When the device does not support the OpenConfig path, vendor-specific CLI commands
+// are applied.
+func PolicyForwardingConfigScale(t *testing.T, dut *ondatra.DUTDevice, sb *gnmi.SetBatch, encapparams OCEncapsulationParams, pf *oc.NetworkInstance_PolicyForwarding, params OcPolicyForwardingParams) *gnmi.SetBatch {
+	t.Helper()
+
+	// Check if the DUT requires CLI-based configuration due to an OpenConfig deviation.
+	if deviations.PolicyForwardingOCUnsupported(dut) {
+		// If deviations exist, apply configuration using vendor-specific CLI commands.
+		switch dut.Vendor() {
+		case ondatra.ARISTA: // Currently supports Arista devices for CLI deviations.
+			PolicyForwardingConfig := new(strings.Builder)
+
+			PolicyForwardingConfig.WriteString("traffic-policies\n")
+
+			for i := 1; i <= encapparams.Count; i++ {
+
+				fmt.Fprintf(PolicyForwardingConfig, `
+    traffic-policy tp_cloud_id_3_%d
+    match setttlv6 ipv6
+       ttl 1
+       !
+       actions
+          count
+          redirect next-hop group 1V6_vlan_3_%d
+          set traffic class 3
+    !
+    match setttlv4 ipv4
+       ttl 1
+       !
+       actions
+          count
+          redirect next-hop group 1V4_vlan_3_%d
+          set traffic class 3
+    !
+    match ipv4-all-default ipv4
+       actions
+          count
+          redirect next-hop group 1V4_vlan_3_%d
+          set traffic class 3
+    !
+    match ipv6-all-default ipv6
+       actions
+          count
+          redirect next-hop group 1V6_vlan_3_%d
+          set traffic class 3
+    !`, i, i, i, i, i)
```

Comment:
Go fmt lib now supports argument indexing so you don't need to pass in multiple copies of the same var.

https://pkg.go.dev/fmt#hdr-Explicit_argument_indexes


```suggestion
				fmt.Fprintf(PolicyForwardingConfig, `
    traffic-policy tp_cloud_id_3_%[1]d
    match setttlv6 ipv6
       ttl 1
       !
       actions
          count
          redirect next-hop group 1V6_vlan_3_%[1]d
          set traffic class 3
    !
    match setttlv4 ipv4
       ttl 1
       !
       actions
          count
          redirect next-hop group 1V4_vlan_3_%[1]d
          set traffic class 3
    !
    match ipv4-all-default ipv4
       actions
          count
          redirect next-hop group 1V4_vlan_3_%[1]d
          set traffic class 3
    !
    match ipv6-all-default ipv6
       actions
          count
          redirect next-hop group 1V6_vlan_3_%[1]d
          set traffic class 3
    !`, i)
```

Reply:
updated as suggested